core/server/common/src/main/java/alluxio/master/journal/raft/JournalStateMachine.java

```diff
@@ -399,8 +399,8 @@ public synchronized void applyJournalEntryCommand(TransactionContext commit) {
    */
   private void applyEntry(JournalEntry entry) {
     Preconditions.checkState(
-        entry.getAllFields().size() <= 1
-            || (entry.getAllFields().size() == 2 && entry.hasSequenceNumber()),
+        entry.getAllFields().size() <= 2
+            || (entry.getAllFields().size() == 3 && entry.hasSequenceNumber()),
```

Comment:
can you help me understand why we needed to update this precondition check?

Reply:
Same with above. Due to additional operationId field.